client/plugins/playbackProfiles/helpers/fmp4AudioFormats.ts

```diff
@@ -25,23 +27,23 @@ export function getSupportedFmp4AudioCodecs(
     codecs.push('mp3');
   }
 
-  if (hasAc3Support(videoTestElement)) {
-    if (hasAc3InHlsSupport(videoTestElement)) {
+  if (hasAc3Support(context, videoTestElement)) {
+    if (hasAc3InHlsSupport(context, videoTestElement)) {
```

Comment:
Passing context around everywhere is starting to remind me of Android development haha.

Reply:
Indeed, but there isn't much else we can do :smile: